hub/core/chunk/chunk_compressed_chunk.py

```diff
@@ -80,10 +80,13 @@ def extend_if_has_space_byte_compression(
             if (
                 len(self.decompressed_bytes) + sample_nbytes  # type: ignore
             ) * self._compression_ratio > self.min_chunk_size:
-                if end is False:
-                    new_decompressed = serialized_sample + self.decompressed_bytes  # type: ignore
-                else:
+
+                decompressed_bytes = self.decompressed_bytes
```

Comment:
is this being used?

Reply:
yes it is used in logic after, this just shows in my changes as I resolved merge conflicts